farmbot.rb

```diff
@@ -1,7 +1,7 @@
 # FarmBot Controller
 
 require_relative 'settings.rb'
-
+require 'pry' if !!ENV['SIMULATE']
```

Comment:
I made a change so that you can run `farmbot.rb` in simulation mode by using an ENV var.

eg:

```bash
SIMULATE=yes ruby farmbot.rb
```